vignettes/taxize_case_study.Rmd

```diff
@@ -0,0 +1,82 @@
+<!--
+%\VignetteEngine{knitr::knitr}
+%\VignetteIndexEntry{Case study - resolving Species Plantarum names}
+%\VignetteEncoding{UTF-8}
+-->
+
+Case study - resolving Species Plantarum names
+==============================================
+
+*Species Plantarum* by Carl Linnaeus originally published in 1753, was the first work to consistently apply binominal names and was the starting point for the naming of plants. It provides useful reference point to see how taxonomic names have changed since their inception. So let's see if we can resolve these names and see if any of the names have changed. 
+
+Firstly, let's see how many of the names still exist in modern species catologues today. We can use ```resolve``` function to query the Global Name Resolver.
+
+```{r resolve, message = FALSE, warning = F, error = FALSE, cache = T}
+library("taxize")
+library("tidyverse")
```

Comment:
wrt `tidyverse` -  I'd rather not introduce more dependencies in this package.  See what's already used in this package here https://github.com/ropensci/taxize/blob/master/DESCRIPTION and you can use any of those if applicable or just base R

Reply:
i know this might be a pain but even Suggests mean more complicated and longer running builds on CI

Reply:
this also mean avoiding pipes here as well

Reply:
okay. removed tidyverse dependencies and used base R.